### DISCUSSION TITLE
Command to early-exit the job and set check conclusion

### DISCUSSION THREAD

**Discussion by jerrygreen** [community/community#121252] [Actions] [answered]
### Select Topic Area

Question

### Body

Make it so that, like in Gitlab CI, `exit 0` would stop the job entirely.

To precisely control this, I'd even like that:

- `exit 0` - preventively ends the job returning success
- `exit 1`, `exit 2`, `exit 3` etc - preventively ends the job returning failure with its exit status
- ...`exit -1` probably? to return into neutral state

More feedback on this issue listed here:

- https://github.com/actions/runner/issues/662

**Comment by grandmas-favorite** [comment, marked as answer]
Hi @jerrygreen!

Thanks for sharing and we appreciate your patience. You can check our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

As a GitHub user, the most you can do is to share your feedback on this Discussion space with the correct tags (include examples), and trust that we are always working to make GitHub as awesome as possible. We can't guarantee that every bit of feedback sees action, but we are paying attention!
